features/202-session-management/api.md

```diff
@@ -11,19 +12,29 @@ interface UserAgent {
     device: string;
 }
 
+// Field value would be empty string if not available
+interface Device {
+    name: string; // device name
+}
+
 interface Session {
     id: string;
     identityID: string;
     createdAt: Date;
     lastAccessedAt: Date;
     createdByIP: string;
     lastAccessedIP: string;
-    createdByUserAgent: UserAgent;
-    lastAccessedUserAgent: UserAgent;
+    userAgent: UserAgent;
+    device: Device?;
     name: string;
     data: object;
 }
 
+interface ExtraSessionInfoConfig {
+    enabled: boolean; // indicate whether extra information is enabled, default false
```

Comment:
Is it a server-side config or client-side config? (btw, I always have this question when I read design spec recently)

I don't think the enabled is needed for server side config, so I hope it is purely an API in the SDK.

The field of `deviceName` could be empty and optional

Reply:
It is client side API. Server side config are in `config.md`.